packages/nx/src/lib.rs

```diff
@@ -1,3 +1,10 @@
+#[cfg(not(target_env = "msvc"))]
+use tikv_jemallocator::Jemalloc;
+
+#[cfg(not(target_env = "msvc"))]
+#[global_allocator]
+static GLOBAL: Jemalloc = Jemalloc;
```

Comment:
Only macos/linux - this essentially optimizes allocations, and increases performance for large allocs from vecs/hashmaps